projects/plugins/social/src/js/components/connection-screen/index.js

```diff
@@ -34,15 +35,18 @@ const ConnectionScreen = () => {
 					<ProductOffer
 						className={ styles.offer }
 						slug={ 'jetpack-social' }
-						title={ 'Jetpack Social' }
+						title={ __( 'Jetpack Social', 'jetpack-social' ) }
 						subTitle={ __(
 							'Share your posts with your social media network and increase your site’s traffic',
 							'jetpack-social'
 						) }
 						features={ [
-							'Connect with Instagram, Facebook, Mastodon, LinkedIn, and Tumblr',
-							'Select the social media to share posts while publishing',
-							'Publish custom messages',
+							__(
+								'Share to Facebook, Instagram, LinkedIn, Mastodon, Tumblr, and Nextdoor',
```

Comment:
Jetpack does not allow using [`Intl.ListFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/format), thus the list has to be hardcoded unlike we did in https://github.com/Automattic/wp-calypso/pull/85766